pluggy/callers.py

```diff
@@ -26,8 +26,12 @@ class HookCallError(Exception):
 
 class _Result(object):
     def __init__(self, result, excinfo):
-        self.result = result
-        self.excinfo = excinfo
+        self._result = result
```

Comment:
Just noticed that this broke [some tests](https://travis-ci.org/pytest-dev/pytest/jobs/274380302) in pytest, so pytest-dev/pytest#2744 is already paying off. 👍